scripts/jenkins/ardana/ansible/roles/input_model_generator/vars/entry-scale-ironic-flat-network.yml

```diff
@@ -0,0 +1,37 @@
+#
+# (c) Copyright 2018 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+
+controllers: 3
+sles_computes: 0
```

Comment:
Cince you don't need SLES and RHEL computes with this input model scenario, you can delete these and replace them with a default value (1 ?) for your `ironic_computes` variable.